src/reducers/User.ts

```diff
@@ -35,6 +37,8 @@ export const user = (state: any, action: any) => {
           token: action.payload.token,
         },
       }
+    case Actions.SignOut.SIGN_OUT:
+      return {}
```

Comment:
Maybe returning defaultState would be better?

Reply:
Fixed this